keras_core/backend/jax/math.py

```diff
@@ -43,3 +43,73 @@ def qr(x, mode="reduced"):
             f"Received: mode={mode}"
         )
     return jax.numpy.linalg.qr(x, mode=mode)
+
+
+def _get_complex_tensor_from_tuple(a):
+    if not isinstance(a, tuple):
```

Comment:
maybe `not isinstance(a, tuple) or len(a) != 2`?

Reply:
also probably support list or tuple, generally interchangeable as input in keras/keras-core